docs/victoriametrics/Cluster-VictoriaMetrics.md

```diff
@@ -1609,13 +1617,13 @@ Below is the output for `/path/to/vmselect -help`:
   -search.logSlowQueryStats duration
      Log query statistics if execution time exceeding this value - see https://docs.victoriametrics.com/victoriametrics/query-stats . Zero disables slow query statistics logging. This flag is available only in VictoriaMetrics enterprise. See https://docs.victoriametrics.com/victoriametrics/enterprise/
   -search.logSlowQueryStatsHeaders array
-     White list of header keys to log for queries exceeding -search.logSlowQueryStats. By default, no headers are logged. This flag is available only in VictoriaMetrics enterprise. See https://docs.victoriametrics.com/victoriametrics/enterprise/
+     White list of header keys to log for queries exceeding -search.logSlowQueryStats. Case insensitive. By default, no headers are logged. This flag is available only in VictoriaMetrics enterprise. See https://docs.victoriametrics.com/victoriametrics/enterprise/
```

Comment:
The description of this flag is unclear. I as a user have no idea what does this flag do after reading its' description. Probably, a link to the corresponding docs with more detailed description will help clarifying the purpose and the usage of this flag.

https://github.com/VictoriaMetrics/VictoriaMetrics/commit/7c0c8cc702fc7c14e6fc1e982fac60af783f3a4f#r164686164